eng/pipelines/runtimelab.yml

```diff
@@ -59,7 +59,7 @@ extends:
             helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
             buildConfig: debug
             platforms:
-#            - linux_x64
+            - linux_x64
```

Comment:
These `linux_x64` jobs only test upstream code - I think we can do with just one to exercise our changes to the build scripts (say, only Release).

Reply:
RIght, have left just Release.